MANIFEST.in

```diff
@@ -1 +1,4 @@
 include pitop/robotics/json/*.json
+include pitop/labs/web/blueprints/**/*.html
+include pitop/labs/web/blueprints/**/*.js
+include pitop/labs/web/blueprints/**/*.css
```

Comment:
All files listed here should move: https://setuptools.readthedocs.io/en/latest/userguide/datafiles.html

Reply:
I'm not clear on how we want to use this and what the advantages would be? It sounds like the files would still be included in the source repo but we'll be able to optionally included them in the built packages, or maybe install them in a particular way?

Is this related to the concerns in #293 about large model data files? In this case these are small source code files that we would want to edit as part of updating the controller features so I don't think the same treatment would apply. There is a minified js file that we might handle differently.

Reply:
According to [here](https://packaging.python.org/guides/distributing-packages-using-setuptools/#manifest-in):
> Note MANIFEST.in does not affect binary distributions such as wheels.

We aren't actively managing this wheel as a recommended installation method, per se, so I think we can ignore this until it becomes an issue for someone.